pylxd/models/instance.py

```diff
@@ -291,20 +291,32 @@ def get(cls, client, name):
         return cls(client, **response.json()["metadata"])
 
     @classmethod
-    def all(cls, client):
+    def all(cls, client, **kwargs):
```

Comment:
As we only expect one optional keyword argument this might be more clear with:
```python3
def all(cls, client, recursion=0)
```

Then you can remove the call to `kwargs.get` on line 302.